### ISSUE TITLE
[DX] Rename PATH_UPDATE_ACTION_* options to PATH_UPDATE_ACTION_*_ALIAS

### ISSUE THREAD

**Issue by klonos** [backdrop/backdrop-issues#1980]
Now, with the Redirect module functionality in core, `PATH_UPDATE_ACTION_DELETE` either deletes the old URL alias, or creates a redirect from the old URL alias to the new one. The name of this option was not updated accordingly (API change?) and could be misleading.

Lets see how badly I'll break things 😅 ...

**Comment by klonos** [comment]
I found a bug where the URL array in the description was not defined properly, resulting in the URL being `/admin/config/search/path/patterns/!url` and fixed that as well.

Please do not merge my PR yet. I need an initial review (once test results come back green) and then I have a few ideas on how to improve this further. Also, my PR for this issue here touches on changes proposed in https://github.com/backdrop/backdrop/pull/1438. So, I would like to see that one merged first and then most likely will replace my PR here with a new one.

**Comment by klonos** [comment]
...alright!. Tests came back green, so I'd like that initial review when someone has the time/energy.

What I fixed besides that URL in the description and the renaming of the options is That I moved the second part of the label for the first radio option ("Delete the old URL alias." / "Replace the old URL alias with a redirect.") inside the `if (module_exists('redirect'))` logic in order to follow the same pattern we have for the respective description.

What I want to do next (once I get the initial review) is to see if there is a way to move the second part of the radio set description (the one that refers to the Redirect module) immediately below the first radio option, because that option is that one that the help text refers to. It does not apply to the other two options. I don't know if this is possible and I would appreciate any pointers (or a "not currently possible" statement, so that I don't waste time on a wild goose chase).

After that, I would like to see if it is possible to pull the value of the "Automatically create redirects when URL aliases are changed." checkbox and adapt the help text in order to be making more sense depending on that value.

**Comment by quicksketch** [comment]
> PATH_UPDATE_ACTION_DELETE either deletes the old URL alias, or creates a redirect from the old URL alias to the new one. The name of this option was not updated accordingly (API change?) and could be misleading.

Yeah that's actually the reason it wasn't renamed, changing it would be considered an API change. So the current PR looks good but we should leave the constants as-is.

**Comment by klonos** [comment]
Fair enough. I thought it most likely would be so.

I still think we should fix the constant names (because it is confusing DX-wise to have a constant called something and actually doing something else), but since that *is* an API change, lets leave it for 2.x.

I'll open other issues and PRs for the other fixes I made that can go in the next 1.4.x, but as I said, I would like https://github.com/backdrop/backdrop/pull/1438 to get in first because it fixes some of the strings I need to change here as well.
